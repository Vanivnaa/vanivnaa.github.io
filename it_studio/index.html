<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
    <header>
        <div class="container">
            <nav class="menu">
                <a href="#" class="menu_item">Главная</a>
                <a href="#" class="menu_item">Наши проекти</a>
                <a href="#" class="menu_item">Услуги</a>
                <a href="#" class="menu_item">Новости</a>
                <a href="#" class="menu_item">Контакти</a>
            </nav>
        </div>
        <div class="line">

        </div>
        <div class="container">
            <div class="offer">
                <p class="offer-subtittle">
                    Передовая IT студия
                </p>
                <h1 class="offer-tittle">
                    Мы создаем легкие решения <br> сложных задач и проблем
                </h1>
                <button class="offer-btn">Наши проекты</button>
            </div>
        </div>
    </header>
    <section>
        <div class="container">
            <div class="sec">
                <h2>Мы создаем мобильные приложения<br>
                для крупных корпораций</h2>
                <p>
                Lorem Ipsum - это текст-"рыба", часто используемый в печати и вэб-дизайне. Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века. 
                </p>
            </div>
            <div class="items">
                    <div class="item">
                        <img src="img/icon.svg" alt="icon">
                        <div class="item_text">
                            <h3>Создание сайтов</h3>
                            <p>Lorem Ipsum - это текст- "рыба", часто используе в печати и вэб-дизайне.</p>
                        </div>
                    </div>
                    <div class="item">
                        <img src="img/icon.svg" alt="icon">
                        <div class="item_text">
                           <h3>Создание сайтов</h3>
                            <p>Lorem Ipsum - это текст- "рыба", часто используе в печати и вэб-дизайне.</p>
                        </div>
                    </div>
                    <div class="item">
                        <img src="img/icon.svg" alt="icon">
                        <div class="item_text">
                          <h3>Создание сайтов</h3>
                            <p>Lorem Ipsum - это текст- "рыба", часто используе в печати и вэб-дизайне.</p>
                        </div>
                    </div>
            </div>
            <div class="item-btn">
                <button class="item__btn">Все услуги</button>
            </div>
        </div>
    </section>
    <section>
        <div class="section-projects">
            <div class="container">
                <div class="block">
                    <div class="blocks">
                        <h4 class="block-tittle">
                            Высокие стандарты разработки
                        </h4>
                        <p class="block-subtittle">
                            Lorem Ipsum - это текст-"рыба", часто используемый в <br> печати и вэб-дизайне. Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века. <br>  стандартной "рыбой" для текстов на латинице с начала <br> XVI века. 
                        </p>
                        <a class="block-link" href="#">Подробнее о компании</a>
                    </div>
                    <img class="desktop" src="img/desktop.jpg" alt="">
                </div>
                <div class="line">

                </div>
                <div class="projects">
                    <div class="block-projects">
                        <img src="img/web_icon.svg" alt="">
                        <h4>125</h4>
                        <p>Our projects</p>
                    </div>
                    <div class="block-projects">
                        <img src="img/web_icon.svg" alt="">
                        <h4>125</h4>
                        <p>Our projects</p>
                    </div>
                    <div class="block-projects">
                        <img src="img/web_icon.svg" alt="">
                        <h4>125</h4>
                        <p>Our projects</p>
                    </div>
                    <div class="block-projects">
                        <img src="img/web_icon.svg" alt="">
                        <h4>125</h4>
                        <p>Our projects</p>
                    </div>
                    <div class="block-projects">
                        <img src="img/web_icon.svg" alt="">
                        <h4>125</h4>
                        <p>Our projects</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section>
    <div class="blue">
        <div class="container">
            <div class="sec_blue">
               <h4>Наши работы, на которых 
                мы специализируемся</h4> 
                <div class="item123">
                    <div class="it1">
                        <img src="img/1.jpg" alt="jpg">
                        <h3><span>01.</span> Исследование и упаковка</h3>
                        <P>Lorem Ipsum - это текст-"рыба", часто используемый в печати и вэб-дизайне. Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века.</P>
                        <a href="#">Подробнее</a>
                    </div>
                    <div class="it2">
                        <img src="img/1.jpg" alt="jpg">
                        <h3><span>01.</span> Исследование и упаковка</h3>
                        <p>Lorem Ipsum - это текст-"рыба", часто используемый в печати и вэб-дизайне. Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века.</p>
                        <a href="#">Подробнее</a>
                    </div>
                    <div class="it3">
                        <img src="img/1.jpg" alt="jpg">
                        <h3><span>01.</span> Исследование и упаковка</h3>
                        <p>Lorem Ipsum - это текст-"рыба", часто используемый в печати и вэб-дизайне. Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века.</p>
                        <a href="#">Подробнее</a>
                    </div>
                </div>
                <div class="line"></div>
            </div>
        </div>  
    </div>
    </section>
    <section>
        
    </section>
</body>
</html>