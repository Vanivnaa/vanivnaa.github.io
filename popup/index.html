<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>PopUp</title>

    <link rel="stylesheet" href="style.css">

</head>
<body>
    <div class="btn">
        <a href="#" id="open_popup">PopUp</a>
    </div>

    <div class="popup" id="popup">
        <div class="popup_container">
            <div class="popup_body" id="popup_body">
                <img src="img/logo.png" alt="ForDog" class="logo">
                <p>Shop our best sellers</p>
                <div class="ph">
                    <img src="img/p1.jpg" alt="goods" class="p1">
                    <img src="img/p2.jpg" alt="goods" class="p2">
                    <img src="img/p3.jpg" alt="goods" class="p3">
                </div>
                <div class="line"></div>
                <div class="btns">
                    <div class="btn_show" id="btn_show">
                        <a href="#">Show me</a>
                    </div>
                    <div class="btn_close" id="btn_close">
                        <a href="#">Close</a>
                    </div>
                </div>
                
            </div>
        </div>
    </div>
    <script src="main.js"></script>
</body>
</html>