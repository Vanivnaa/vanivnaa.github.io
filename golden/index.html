<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Golden</title>
    <link rel="stylesheet" href="css\main.css">
</head>
<body>
    <header>
        <div class="container">
			<div class="heading clearfix">
				<img src="img/logo.png" alt="Golden" class="logo">
				<nav>
					<ul class="menu">
						<li>
							<a href="#">Home</a>
						</li>
						<li>
							<a href="#">Services</a>
						</li>
						<li>
							<a href="#">Portfolio</a>
						</li>
						<li>
							<a href="#">About</a>
						</li>
						<li>
							<a href="#">Contact</a>
						</li>
					</ul>
				</nav>
			</div>
			<div class="titles">
				<div class="titles__first">
					Welcome To Our Studio!
				</div>
				<h1>
					It’s nice to Meet you
				</h1>
			</div>
			<a class="button" href="#">Tell Me More</a>
		</div>
    </header>

    <section>
        <div class="container">
			<div class="title">
				<h2>Services</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="services">
				<div class="services_">
					<img src="img/icon1.png" alt="Послуга">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services_">
					<img src="img/icon2.png" alt="Послуга">
					<h3>Responsive Web</h3>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services_">
					<img src="img/icon3.png" alt="Послуга">
					<h3>Web Security</h3>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>	

			</div>
        </div>
        
    </section>
    <section id="portfolio" class="portfolio">
        <div class="container">
			<div class="title">
				<h2>Our portfolio</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
			</div>
			<div class="img_portf">
				<img src="img/img1.jpg" alt="portfolio">
				<img src="img/img2.jpg" alt="portfolio">
				<img src="img/img3.jpg" alt="portfolio">
			</div>
        </div>
        
    </section>
     <section>
        <div class="container">
			<div class="title">
				<h2>About us</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="about">
				<div class="ab_1">
					<div class="descr">
						<h4>July 2010</h4>
						<h3>Our Humble Beginnings</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
		
					<div class="ab1_img">
						<img src="img/about1.jpg" alt="about">
					</div>
				</div>

				<div class="ab_2">
					<div class="ab2_img">
						<img src="img/about2.jpg" alt="about">
					</div>

					<div class="descr">
						<h4>January 2011</h4>
						<h3>Facing Startup Battles</h3>
						<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit aenean 
							rhoncus posuere odio in tincidunt.</p>
					</div>
				</div>

				<div class="ab_3">
					<div class="descr">
						<h4>December 2012</h4>
						<h3>Enter The Dark Days</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim.</p>
					</div>
		
					<div class="ab3_img">
						<img src="img/about3.jpg" alt="about">
					</div>
				</div>

				<div class="ab_4">
					<div class="ab4_img">
						<img src="img/about4.jpg" alt="about">
					</div>

					<div class="descr">
						<h4>february 20141</h4>
						<h3>Our Triumph</h3>
						<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit aenean.</p>
					</div>
				</div>	
			</div>
        </div>
        
    </section>
    <footer>
        <div class="container">
			<div class="text">
			<p>© Copyright 2014 FreebiesXpress.com</p>	
			</div>
        </div>
        
    </footer>
  
</body>
</html>